frontend/public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, viewport-fit=cover" />
-    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="mobile-web-app-capable" content="yes" />
```

Comment:
I can imagine this is deprecated from Google / Chrome perspective if apple introduced this and the apple-less variant is there now for them to support. Knowing Apple I could imagine this is still the tag for Safari though. Hard to test for me, but I'd actually assume we may need both to be compatible at both sides.

In a (random) example like this you see this double usage as well: https://gist.github.com/hal0gen/5852bd9db240c477f20c#file-_mobile-ready-web-app-html-L12-L15

I suppose we definitely should not forget testing on apple devices

Reply:
Hmm... Yes, there seems to be something weird going on here. We'll have to try it. It's only since April or May that Apple really supports the PWA manifest, so it might be that the tag needs to be duplicated, or it might be that it needs to be removed altogether.